packages/react/src/components/Search/Search-story.js

```diff
@@ -61,6 +69,34 @@ export default {
 
 export const Default = () => <Search {...props()} id="search-1" />;
 
+export const SizeStory = () => (
+  <div>
+    <div>
+      <h3>Feature Flags: DISABLED</h3>
+      <br />
+      <br />
+      <Search placeholder="sm" size="sm" id="search-1" />
+      <br />
+      <Search placeholder="lg" size="lg" id="search-1" />
+      <br />
+      <Search placeholder="xl" size="xl" id="search-1" />
+      <br />
+    </div>
+    <br />
+    <br />
+    <FeatureFlags flags={{ 'enable-2021-release': true }}>
```

Comment:
Do you think it'd be worth toggling when the feature flag stories get shared? Maybe something like:

```jsx
import { enabled } from '@carbon/feature-flags';

export default {
  title: '...',
  // ...
  excludeStories: [
    {
      name: 'StoryName',
      enabled: enabled('enable-2021-release'),
    },
  ].filter(story => {
    return story.enabled;
  }).map(story => {
    return story.name;
  }),
};
```

Reply:
Yeah that might be a good idea, I was just going to remove the feature flag story, but I can add it as a separate story that only shows up when the feature flag is toggled at the top level. 

Do we currently have any way to flip the switch at the storybook level?

Reply:
Hmm, not able to get this working, keeps causing build failures. Were you able to get this working on your end?